app.py

```diff
@@ -126,16 +141,21 @@ def get_eventbrite_events(group_list):
     token = config.get('eventbrite', 'token')
     events = []
 
+    # Number of days to allow for past events
+    start_days = config.get('past_events', 'past_days')
```

Comment:
@ramonaspence  This is more of a readability thing, but how about we call this days_in_the_past  ?

I was reviewing it and got confused with start_days and start_date.